account/transaction.go

```diff
@@ -18,23 +18,15 @@ import (
 //   - ctx: The context.Context for the request.
 //   - functionCalls: A slice of rpc.InvokeFunctionCall representing the function calls for the transaction, allowing either single or
 //     multiple function calls in the same transaction.
-//   - multiplier: A safety factor for fee estimation that helps prevent transaction failures due to
-//     fee fluctuations. It multiplies both the max amount and max price per unit by this value.
-//     A value of 1.5 (50% buffer) is recommended to balance between transaction success rate and
-//     avoiding excessive fees. Higher values provide more safety margin but may result in overpayment.
-//   - withQueryBitVersion: A boolean flag indicating whether the transaction version should have the query bit when estimating fees.
-//     If true, the transaction version will be rpc.TransactionV3WithQueryBit (0x100000000000000000000000000000003).
-//     If false, the transaction version will be rpc.TransactionV3 (0x3).
-//     In case of doubt, set to 'false'.
+//   - opts: options for building/estimating the transaction. See more info in the TxnOptions type description.
```

Comment:
This is very subjective but I would remove the second sentence because anyone looking at the code would do this implicitly and it's harder to maintain overall, but this is also me being very extra nitpicky, so feel free to dismiss.